server/storage/rethinkdb.go

```diff
@@ -0,0 +1,205 @@
+package storage
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/dancannon/gorethink"
+	"github.com/docker/notary/storage/rethinkdb"
+)
+
+// RDBTUFFile is a tuf file record
+type RDBTUFFile struct {
```

Comment:
Oh definitely,I just meant the `rethink_models` file (which are right now I think table definitions)